app/src/main/java/org/wikipedia/util/StringUtil.kt

```diff
@@ -88,6 +88,11 @@ object StringUtil {
         return text.replace("<cite.*?>".toRegex(), "").replace("</cite>".toRegex(), "")
     }
 
+    @JvmStatic
+    fun sanitizeAbuseFilterCode(code: String): String {
+        return code.replace("⧼", "").replace("⧽", "")
```

Comment:
Should we use `toRegex()` here? `code.replace("<|>".toRegex(), "")`.

Reply:
yep! also added a test.